src/components/modals/ActionCreatorEditor.tsx

```diff
@@ -1868,7 +1897,8 @@ const mapStateToProps = (state: State, ownProps: any) => {
         actions: state.actions,
         trainDialogs: state.trainDialogs,
         botInfo: state.bot.botInfo,
-        browserId: state.bot.browserId
+        browserId: state.bot.browserId,
+        settings: state.settings
```

Comment:
nit: prefer trailing commas on everything but array items

Reply:
Not sure I follow your comment ?